tests/integration/fake-sources/fake.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  country: 'iso1:US',
+  state: 'CA',
+  maintainers: [],
+  friendly: {
+    name: 'Fake source',
+    url: 'rick-roll-url'
```

Comment:
I mean, come on, put the actual URL in there at least! 🎤

Reply:
Done!